library/core/tests/lib.rs

```diff
@@ -77,6 +78,9 @@
 #![feature(iter_repeat_n)]
 #![feature(iterator_try_collect)]
 #![feature(iterator_try_reduce)]
+#![feature(const_ip)]
+#![feature(const_ipv4)]
+#![feature(const_ipv6)]
```

Comment:
I am very surprised that this does not need the `ip_in_core` feature? miri-test-libstd is [failing due to that feature missing somewhere](https://github.com/rust-lang/miri-test-libstd/actions/runs/4288908148/jobs/7471415401).

Reply:
I guess because `core` itself does not use anything from `core::net` (which is the unstable module). Submodules of `core::net` uses parts of itself, but I guess if you are inside an unstable module you can freely use the stuff in it?

Reply:
This is a different crate though, it is the coretest crate, not the core crate.

Reply:
Ah, sorry. I misread the path! Yeah, that's very strange. But I see you have a PR up for fixing it :+1:

Reply:
Yeah it fixes miri-test-libstd. But still I am worried that the fact that it builds here without the feature flag might be indicative of a more fundamental problem -- as mentioned elsewhere, historically reexport stability did not work properly.